frontend/src/components/pomodoroview/PomodoroView.jsx

```diff
@@ -1,27 +1,29 @@
-import React, { useState, useContext, createContext } from "react";
+
+import React, { useState, useContext, createContext} from "react";
 import { CountdownCircleTimer } from "react-countdown-circle-timer";
 import store from "../../redux-store";
 import SubmitRecordService from "../../services/SubmitRecordService";
+import './PomodoroView.css'
 
-// css modules
 
+// css modules
 const timerWrapper = {
   display: "flex",
   justifyContent: "center",
 };
 
 const workTimerProps = {
-  colors: [["#49DD78", 0.33], ["#6C6CE0", 0.66], ["#E71A1A"]],
+  colors: [["#007bff", 0.33], ["#6C6CE0", 0.66], ["#E71A1A"]],
```

Comment:
It is a design to change the color from Green(safe) to Red(urgent). So maybe change it back to green and change the button color also ? or find a nicer green if you want.